ClientApp/viking-vault/src/UserDashboard/AutomaticPaymentList.tsx

```diff
@@ -2,12 +2,19 @@ import  React  from 'react';
 import {constants} from "../Resources/Constants";
 import './ViewAutomaticPayments.css'
 import ActivatePaymentToggle from "../AdminDashboard/ActivatePaymentToggle"
+import {CreateAutomaticPaymentForm} from "./CreateAutomaticPaymentModal"
+import add_payment from "../Resources/images/add_payment.png"
```

Comment:
Please keep this variable's naming style consistent to the rest of the application, which uses `camelCase`, also I think that naming this `addPaymentImg` would be more descriptive